backend/src/test/java/com/github/nickklock/backend/services/SpotServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.github.nickklock.backend.services;
+
+import com.github.nickklock.backend.models.Position;
+import com.github.nickklock.backend.models.Spot;
+import com.github.nickklock.backend.models.SpotRequest;
+import com.github.nickklock.backend.repos.SpotRepo;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class SpotServiceTest {
+
+    SpotRepo spotRepo = mock(SpotRepo.class);
+    IdService idService = mock(IdService.class);
+    SpotService spotService = new SpotService(spotRepo, idService);
+
+    @Test
+    void add_expect_given_and_result_equals_verify_shoprepo() {
+        SpotRequest spotRequest = new SpotRequest( "test", new ArrayList<>(),
+                new ArrayList<>(), new ArrayList<>(), new ArrayList<>(), new ArrayList<>(),
+                new ArrayList<>(), new ArrayList<>(), new ArrayList<>(),
+                1, new Position(0, 0),"Yes");
```

Comment:
Some additional FYI: You could write a reusable helper method to create this kind of repeating dummy data.

Reply:
Will do this once it becomes more often